src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -103,35 +79,10 @@ const PagesAndResources = ({ courseId, intl }) => {
             <DiscussionsSettings courseId={courseId} />
           </PageRoute>
 
-          <PageRoute
-            path={[
-              `${path}/xpert-unit-summary/settings`,
-            ]}
-          >
-            <XpertUnitSummarySettings courseId={courseId} />
-          </PageRoute>
-
           <PageRoute path={`${path}/:appId/settings`}>
             {
-              ({ match, history }) => {
-                const SettingsComponent = React.lazy(async () => {
```

Comment:
Commentary: There was a some kind of bug with this line, where the `React.lazy` was actually getting re-evaluated many times, and the `useEffect` of the Xpert Unit Summary plugin was getting called over and over, hammering the backend with requests to populate the redux state. This wasn't actually affecting the current MFE because it was hard-coded to load Xpert Unit Summary settings differently (see removed lines above). But when I changed it to be loaded like the other plugins, this bug occurred. The fix was easy, however: I moved it to a component and put it behind a `useMemo` hook, so it would only load the state once.